website/docs/cloud-docs/api-docs/data-retention-policies.mdx

```diff
@@ -0,0 +1,127 @@
+---
+page_title: Data Retention Policies - API Docs - Terraform Cloud
```

Comment:
Out of curiosity: Is it possible to remove "Terraform Cloud" from the page title here since this is a TFE-only feature, or is that page title breadcrumb necessary regardless?